Dockerfile.armhf

```diff
@@ -0,0 +1,86 @@
+FROM golang:1.13-alpine AS build
+
+ENV \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=arm \
+    GOARM=7 \
+    GOVERALLS=0.0.5 \
+    GOLANGCI=1.23.6 \
+    STATIK=0.1.6
+
+RUN \
+    apk add --no-cache --update tzdata git bash curl && \
+    cp /usr/share/zoneinfo/America/Chicago /etc/localtime && \
+    rm -rf /var/cache/apk/*
+
+RUN \
+    go version && \
+    go get -u -v github.com/golang/dep/cmd/dep && \
+    go get -u -v github.com/kardianos/govendor && \
+    go get -u -v github.com/jteeuwen/go-bindata/... && \
+    go get -u -v github.com/stretchr/testify && \
+    go get -u -v github.com/vektra/mockery/.../
+
+RUN \
+    go get -u -v github.com/golangci/golangci-lint/cmd/golangci-lint && \
+    cd /go/src/github.com/golangci/golangci-lint && \
+    git checkout v${GOLANGCI} && \
+    cd /go/src/github.com/golangci/golangci-lint/cmd/golangci-lint && \
+    go install -ldflags "-X 'main.version=$(git describe --tags)' -X 'main.commi                                   t=$(git rev-parse --short HEAD)' -X 'main.date=$(date)'" && \
```

Comment:
I think those extra spaces here not intentionally, typo?

Reply:
sorry, fixed that.